test/azul_test_case.py

```diff
@@ -1,10 +1,49 @@
-import os
-from unittest import TestCase, mock
+from unittest import TestCase
 
 import boto3.session
 
 
-class AzulTestCase(TestCase):
+class SafeTestCase(TestCase):
```

Comment:
```suggestion
class AlwaysTearDownTestCase(TestCase):
```

I find "Safe" a little bit presumptuous. Whether one approach is "safer" than the other is debatable and depends on the circumstances. I prefer to be more specific because the verbosity of the name isn't hurting in this case.